gradle/verification-metadata.xml

```diff
@@ -1406,6 +1406,16 @@
             <sha256 value="3dcf5c33adddec2655f85ea2912da294fc0ea210cb1dd032b5278d78a5a0d805" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="ethereum" name="execution-spec-tests" version="0.2.4">
+         <artifact name="execution-spec-tests-0.2.4-fixtures.tar.gz">
```

Comment:
Does it matter if we leave the older versions in here?

Reply:
file bloat. will take them out